### PR TITLE
OSX system font

### PR DESCRIPTION
**Pull request by tallytalwar**
First step to debug and fix AAT system fonts with harfbuzz. This work coming in another PR.

OSX platform does not use bundled fallback fonts but relies on the system fonts for fallback.

### CODE REVIEW

File: platforms/common/TGFontConverter.h [hunk 1]
```diff
@@ -0,0 +1,24 @@
+//
```
Comment (karimnaaji) [inline platforms/common/TGFontConverter.h]: We can remove these files from `platforms/ios/src/TangramMap/` now.

Comment (tallytalwar) [inline platforms/common/TGFontConverter.h]: Yup will do. And yes it has no dependency on harfbuzz updates. I had just thought, that PR will go in first :D.

Also we should wait on `0.8.0` to be released before we merge this or any other PR in.

File: platforms/osx/src/osxPlatform.mm [hunk 2]
```diff
@@ -93,18 +96,125 @@ void initGLExtensions() {
...
+    CGFontRelease(fontRef);
+
+    if (data.empty()) {
+        LOG("CoreGraphics font failed to decode");
+    }
+
+    return data;
+}
+
+bool allowedFamily(NSString* familyName) {
```
Comment (karimnaaji) [inline platforms/osx/src/osxPlatform.mm]: We could share the list of allowed families between iOS and OSX if those are the same.

Comment (tallytalwar) [inline platforms/osx/src/osxPlatform.mm]: Yeah I can move this to a common file and share the code.

Comment (tallytalwar) [reply platforms/osx/src/osxPlatform.mm]: I will take care of this in the coretext usage work I am doing.

**Pull request merged**